static/styles/zulip.scss

```diff
@@ -1411,6 +1411,7 @@ div.focused_table {
     vertical-align: top;
     border-radius: 4px;
     overflow: hidden;
+    background-color: hsl(0, 0%, 90%);
```

Comment:
This breaks the transparency of transparent avatars.

Reply:
Yeah, I noticed. Removing the background for loaded images isn't clean since we need to use the load event on the image element and since it doesn't bubble we have to loop over them and handle adding the listener when adding new events. So either this PR or removing the background on load isn't worth the effort.